ddtrace/tracer/option.go

```diff
@@ -1191,6 +1191,43 @@ func AnalyticsRate(rate float64) StartSpanOption {
 	return Tag(ext.EventSampleRate, rate)
 }
 
+// WithConfig merges the given StartSpanConfig into the one used to start the span.
+// It is useful when you want to set a common base config, reducing the number of function calls in hot loops.
+func WithConfig(cfg ddtrace.StartSpanConfig) StartSpanOption {
```

Comment:
Have we considered naming it `WithSpanConfig`  or `WithStartSpanConfig` ?
Asking because the function is defined in the `tracer` package and would be called `tracer.WithConfig` which implies that it's a tracer (not a span) config.

Reply:
You are right, I'll change this to `WithStartSpanConfig`.